test_requirements.txt

```diff
@@ -4,7 +4,7 @@ pytest-mock
 pylint
 black
 scipy
-dash[testing]
+dash[testing]==2.5.1
```

Comment:
Why do we need to pin?

Reply:
for some reason (i'm not sure why, i don't understand the inner working of pip orhow it resolves dependencies) the jenkins run was installing dash 2.4, and that one didn't have the method `wait_for_class_to_equal` that i needed for testing... so i just bumped it to a higher version.

but maybe, is there a notation like `>=2.5` that can be used? gonna google that real quick...

Reply:
fixed, it now says `>=2.5.1`